create-src-tarball.sh

```diff
@@ -0,0 +1,5 @@
+VER=$(git describe --tags --match "v[0-9]*" --abbrev=0 | sed "s/^v//")
+
+git archive --format tar.gz \
+  --prefix will-crash-${VER}/ \
+  --output will-crash-${VER}.tar.gz v${VER}
```

Comment:
When we are using tito, then this should be `tito build --tgz` or `tito build --test --tgz`. Depends whether we want tarball from last tag or commit.

Reply:
Oh well, I did suspect there must be a better way.